spec/rack_attack_throttle_spec.rb

```diff
@@ -148,3 +148,38 @@
     end
   end
 end
+
+describe 'Rack::Attack.throttle with case sensivity option for discriminator' do
+  before do
+    @period = 60
+    Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
+    Rack::Attack.throttle('logins/email', limit: 2, period: @period) do |req|
+      if req.path == '/login' && req.post?
+        req.params['email']
+      end
+    end
+  end
+
+  it 'should not differentiate requests when discriminator_normalizer is enabled' do
+    post '/login', email: 'person@example.com'
+    post '/login', email: 'PERSON@example.com'
```

Comment:
Consider adding a few more email variants that will give confidence surrounding and inner, single+multiple consecutive whitespaces are normalized.